fairlearn/datasets/_fetch_acs_income.py

```diff
@@ -0,0 +1,160 @@
+# Copyright (c) Microsoft Corporation and Fairlearn contributors.
+# Licensed under the MIT License.
+
+import pathlib
+
+import numpy as np
+import pandas as pd
+from sklearn.datasets import fetch_openml
+from ._constants import _DOWNLOAD_DIRECTORY_NAME
+
+
+def fetch_acs_income(*, cache=True, data_home=None,
+                     as_frame=False, return_X_y=False,
+                     states=None,
+                     ):
+    """Load the ACS Income dataset.
```

Comment:
We should really refer to the relevant user guide item here.

Reply:
Added a reference!